helm_deploy/values-prod.yaml

```diff
@@ -4,6 +4,7 @@
 generic-service:
   ingress:
     host: hmpps-tier.hmpps.service.justice.gov.uk
+    contextColour: green
```

Comment:
they should all now be green so could you set it https://github.com/ministryofjustice/hmpps-tier/blob/main/helm_deploy/hmpps-tier/values.yaml#L12 block?